act-rules-format.bs

```diff
@@ -245,183 +259,212 @@ For other aspects that are not well defined, an ACT Rule MUST include either a d
+The applicability of a composite rule is defined as the union of all the applicability sections of rules in the [input rules](#input-rules). This includes [=atomic rules=], and [=composite rules=] which themselves have a union applicability. Rule authors MAY ommit a description of the applicability for composite rules. This can be useful if it is difficult to express the combined applicability in plain language. If the composite rule includes applicability, it MUST be the union of all the applicability in the [input rules](#input-rules).
 
-Note that atomic rules in a composite rule MAY have different applicability. Because of this, not every element applicable within the composite rule is tested by every atomic rule.
+Note that input rules in a composite rule MAY have different applicability. Because of this, not every test target applicable within the composite rule is tested by every input rule.
 
 <div class=example>
-  <p><strong>Example</strong>: A composite rule about `img` elements uses results from atomic rules that have the following applicability:</p>
+  <p>**Example:** A composite rule about `img` elements uses results from rules that have the following applicability:</p>
   <ul>
-    <li><strong>Atomic rule1</strong>: All `img` element with an `alt` attribute</li>
-    <li><strong>Atomic rule1</strong>: All `img` element without an `alt` attribute</li>
+    <li>**Rule 1:** All `img` element _with_ an `alt` attribute</li>
+    <li>**Rule 2:** All `img` element _without_ an `alt` attribute</li>
   </ul>
-
-  <p>The applicability of the composite rule combines the applicability of both atomic rules. This becomes:</p>
+  <p>The applicability of the composite rule combines the applicability of both rules. This becomes:</p>
   <blockquote>
     <p>All `img` elements.</p>
   </blockquote>
 </div>
 
 
 Expectations {#expectations}
-==============================
+----------------------------
+
+An ACT Rule MUST contain one or more expectations. The expectations describe what the requirements are for [=test targets=] derived from the [applicability](#applicability). An expectation is an assertion about all [=test targets=] that is true if the [=test subject=] satisfies the [=accessibility requirement=]. When all expectations are true for a test subject, the test subject `passed` the rule. If one or more expectations are false, the test subject `failed` the rule. If there are no test targets, the [=outcome=] for the test subject is `inapplicable`.
```

Comment:
Are we certain about "that is true if the test subject satisfies the accessibility requirement"? In think that individual expectations in individual atomic rules may be not met in a composite rule, while the test subject overall satisfies the accessibility requirement.

Reply:
Following the ACT TF discussion on 31 January 2019, I think we should rather say:

> An expectation is an assertion about all test targets or about the collective set of test targets.